lib/class-wp-theme-json-gutenberg.php

```diff
@@ -2253,43 +2253,37 @@ protected static function flatten_tree( $tree, $prefix = '', $token = '--' ) {
 	 * @return array  Returns the modified $declarations.
 	 */
 	protected static function compute_style_properties( $styles, $settings = array(), $properties = null, $theme_json = null, $selector = null, $use_root_padding = null ) {
-		if ( null === $properties ) {
-			$properties = static::PROPERTIES_METADATA;
-		}
-
-		$declarations = array();
 		if ( empty( $styles ) ) {
-			return $declarations;
+			return array();
 		}
 
+		if ( null === $properties ) {
+			$properties = static::PROPERTIES_METADATA;
+		}
+		$declarations             = array();
 		$root_variable_duplicates = array();
+		$string_length            = strlen( '--wp--style--root--' );
```

Comment:
This variable could be more descriptive. Would need to update usages below.

```suggestion
		$root_style_length        = strlen( '--wp--style--root--' );
```